tests/downloader/test_base.py

```diff
@@ -42,6 +47,49 @@ def test_run_does_not_call_before_build_after_with_resume():
     assert not downloader.after.called
 
 
+def test_run_skip_download(downloader):
+    downloader.download = mock.MagicMock(return_value=True)
+    downloader._is_download_skipped = lambda *args, **kwargs: True
+
+    mock_work = get_mock_work()
+    with pytest.helpers.generate_tmp_path() as tmp_path:
+        tmp_path_str = str(tmp_path)
+        os.mkdir(tmp_path_str)
```

Comment:
You are casting `Path` to `str` where it is unnecessary. Better: `tmp_path.mkdir()`.